### PR TITLE
Added light versions of funder logos

### PR DESCRIPTION
**Pull request by alexandercyu**
Added reverse/white logos of Government of BC, Ouranos, Canadian Foundation for Innovation, and Concordia.

~~Concordia's white logo is actually a drawn svg.  I put the code into an html file for now.~~

Concordia logo made into svg image.

### CODE REVIEW

File: static/images/logos/funders/reverse/concordia-logo.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
+
+<html>
```
Comment (mishaschwartz) [inline static/images/logos/funders/reverse/concordia-logo.html:2]: This is an entire html block ... this can't be right?

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
With regards to all file names: https://github.com/DACCS-Climate/marble-website/issues/85

File: static/images/logos/funders/reverse/concordia-logo.html (hunk 1 above)
Comment (alexandercyu) [reply static/images/logos/funders/reverse/concordia-logo.html:2]: I'm experimenting turning this into an svg image by using Inkscape's xml editor.

Comment (alexandercyu) [reply static/images/logos/funders/reverse/concordia-logo.html]: Concordia logo converted to svg.

**Comment by alexandercyu** [comment]
> With regards to all file names: #85

Renamed British Columbia logos.

**Pull request merged**